LuckyDragon/app/src/main/java/com/example/luckydragon/Activities/ViewProfilesActivity.java

```diff
@@ -55,4 +65,10 @@ public void notifyAdapter() {
 
     }
 
+    @Override
```

Comment:
instead of on resume, make userlist have an listener for whenever the dataset changes and then it updates itself and notifies observers

Reply:
Done